src/lxc/pam/meson.build

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+pam_cgfs_includes = include_directories(
+    '.',
+    '../',
+    '../cgroups',
+    '../../include',
+    '../../../')
+
+pam_cgfs_sources = files(
+    'pam_cgfs.c',
+    '../file_utils.c',
+    '../file_utils.h',
+    '../macro.h',
+    '../memory_utils.h',
+    '../string_utils.c',
+    '../string_utils.h') + include_sources
+
+pam_cgfs_common_dependencies = declare_dependency(
+    include_directories: pam_cgfs_includes,
+    sources: pam_cgfs_sources,
+    dependencies: [
+        pkgconfig_libs,
+    ])
+
+pam_cgfs = shared_module(
+    'pam_cgfs',
+    pam_cgfs_sources,
+    dependencies: pam_cgfs_common_dependencies,
```

Comment:
I'm a bit curious... what's the point of this declare_dependency? You use it in exactly one place, and add each source file twice to the shared_module.